validator/db/db.go

```diff
@@ -68,6 +68,7 @@ func NewKVStore(dirPath string, pubKeys [][48]byte) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	log.Infof(".db & validator keys dir is: %v", dirPath)
```

Comment:
This is not correct, the validator keys are not in this directory. You will need another log to declare where they are. Look at how the `Checking DB` log is put out and mirror that for this one.